lib/ansiblelint/file_utils.py

```diff
@@ -1,8 +1,19 @@
 """Utility functions related to file operations."""
 import os
+import sys
+from pathlib import Path
+from typing import Optional, Union
```

Comment:
The idiom for imports that are unnecessary in runtime is conditional imports of those things:
```suggestion
from typing import Optional, TYPE_CHECKING, Union

if TYPE_CHECKING:
    from pathlib import Path
```